app/controllers/SeriesDetailsController.scala

```diff
@@ -0,0 +1,32 @@
+package controllers
+
+import javax.inject.{Inject, Singleton}
+import org.pac4j.core.profile.CommonProfile
+import org.pac4j.play.scala.{Security, SecurityComponents}
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, Request}
+
+@Singleton
+class SeriesDetailsController @Inject()(val controllerComponents: SecurityComponents)
+                                         extends Security[CommonProfile] with I18nSupport  {
+
+  val form = Form(
+    mapping(
+      "seriesNo" -> number
```

Comment:
Maybe call it seriesId or seriesName rather than seriesNo, since it's not a number? How do archivists refer to it?